### DISCUSSION TITLE
Data visualization (stress, strain, etc) for each grain

### DISCUSSION THREAD

**Discussion by shahul31** [neperfepx/FEPX#84] [Q&A] [answered]
Hello,

Could you please help! Is it possible to visualize the postprocessing results for each grain instead of elementwise, for example - data postprocessing of stress or strain is generated elementwise where the color scheme varies for stress or strain variation, can the same applied for grainwise instead (like given for grain diameter (https://neper.info/tutorials/visualize_tessellation.html#scene-settings). It would be useful to analyze the results for each grain.

**Comment by mkasemer** [comment, marked as answer]
This is a question for the Neper discussion forum, not for the FEPX discussion forum.

**Comment by shahul31** [comment]
Thanks, I will repost at Neper
